### PR TITLE
User keymap for KBDfans KBD67 MKII RGB (hot-swap)

### PR DESCRIPTION
**Pull request by dnsnrk**
## Description

This request comprises a user keymap for the  KBDfans KBD67 MKII RGB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	HSV hsv = rgb_matrix_config.hsv;
+	if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
+	    HSV hsv_inv_hue = {hsv.h + 128, hsv.s, hsv.v};
+	    set_hsv_at(hsv_inv_hue, 30);
+	}
+	// indicate which higher layer is currently active on which key
+	HSV hsv1 = {hsv.h - 64, hsv.s, hsv.v};
+	HSV hsv2 = {hsv.h + 64, hsv.s, hsv.v};
+	// except when we are in any animated LED mode
+	if (rgb_matrix_config.mode < RGB_MATRIX_BREATHING || rgb_matrix_config.mode > RGB_MATRIX_JELLYBEAN_RAINDROPS) {
```
Comment (xplusplus) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: nit: please run clang-tidy on this file

Comment (drashna) [reply keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: @xplusplus we do have clang-format set up, and you can easily run `bin/qmk cformat /path/to/file` for that.  

But yeah, it looks like there is a mix of taps and spaces here that are causing some "fun" indentation rendering issues.

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#include QMK_KEYBOARD_H
+
+#define RC_ENT RCTL_T(KC_ENT)
+#define LC_CPS LCTL_T(KC_CAPS)
+#define KC_VDN KC__VOLDOWN
+#define KC_VUP KC__VOLUP
+
+#define _LAYER0 0
+#define _LAYER1 1
+#define _LAYER2 2
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: You may want to use an enum here instead, as it's easier to maintain long term.

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define RC_ENT RCTL_T(KC_ENT)
+#define LC_CPS LCTL_T(KC_CAPS)
+#define KC_VDN KC__VOLDOWN
+#define KC_VUP KC__VOLUP
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: Should be able to use `KC_VOLU` and `KC_VOLD` here, as long as `EXTRAKEYS` is enabled. 
https://docs.qmk.fm/#/keycodes_basic?id=media-keys

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c (hunk 1 above)
Comment (dnsnrk) [reply keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: I think figured how the tabs got in an fixed my environment. `qmk cformat` was a tad too rude to custom alignments.

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c (hunk 2 above)
Comment (dnsnrk) [reply keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: good point

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c (hunk 3 above)
Comment (dnsnrk) [reply keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c]: yep, they work equally well. Supposedly, I used the first codes appearing in the documentation without further scrolling down.

**Pull request merged**